opentelemetry/proto/logs/v1/logs.proto

```diff
@@ -0,0 +1,130 @@
+// Copyright 2020, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package opentelemetry.proto.logs.v1;
+
+import "opentelemetry/proto/common/v1/common.proto";
+import "opentelemetry/proto/resource/v1/resource.proto";
+
+option java_multiple_files = true;
+option java_package = "io.opentelemetry.proto.logs.v1";
+option java_outer_classname = "LogsProto";
+option go_package = "github.com/open-telemetry/opentelemetry-proto/gen/go/logs/v1";
+
+// A collection of InstrumentationLibraryLogs from a Resource.
+message ResourceLogs {
+    // The resource for the logs in this message.
+    // If this field is not set then no resource info is known.
+    opentelemetry.proto.resource.v1.Resource resource = 1;
+
+    // A list of InstrumentationLibraryLogs that originate from a resource.
+    repeated InstrumentationLibraryLogs instrumentation_library_spans = 2;
+}
+
+// A collection of Logs produced by an InstrumentationLibrary.
+message InstrumentationLibraryLogs {
+    // The instrumentation library information for the logs in this message.
+    // If this field is not set then no library info is known.
+    opentelemetry.proto.common.v1.InstrumentationLibrary instrumentation_library = 1;
+
+    // A list of log records.
+    repeated LogRecord logs = 2;
+}
+
+// Possible values for LogRecord.SeverityNumber.
+enum SeverityNumber {
+    UNDEFINED_SEVERITY_NUMBER = 0;
+    TRACE  = 1;
+    TRACE2 = 2;
+    TRACE3 = 3;
+    TRACE4 = 4;
+    DEBUG  = 5;
+    DEBUG2 = 6;
+    DEBUG3 = 7;
+    DEBUG4 = 8;
+    INFO   = 9;
+    INFO2  = 10;
+    INFO3  = 11;
+    INFO4  = 12;
+    WARN   = 13;
+    WARN2  = 14;
+    WARN3  = 15;
+    WARN4  = 16;
+    ERROR  = 17;
+    ERROR2 = 18;
+    ERROR3 = 19;
+    ERROR4 = 20;
+    FATAL  = 21;
+    FATAL2 = 22;
+    FATAL3 = 23;
+    FATAL4 = 24;
+}
+
+// Masks for LogRecord.flags field.
+enum LogRecordFlags {
+    UNDEFINED_LOG_RECORD_FLAG = 0;
+    TRACE_FLAGS_MASK = 0x000000FF;
+}
+
+// A log record according to OpenTelemetry Log Data Model:
+// https://github.com/open-telemetry/oteps/blob/master/text/logs/0097-log-data-model.md
+message LogRecord {
+    // time_unix_nano is the time when the event occurred.
```

Comment:
There's a mismatch between the documented name `time_unix_nano` and the actual name `time_unixnano` here.

Reply:
Fixed, thank you. `time_unix_nano` is the right name, uniform with span field naming.